web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakJson.kt

```diff
@@ -55,16 +56,37 @@ internal data class SakJson(
                 )
             },
             utbetalingerKanStansesEllerGjenopptas = utbetalingstidslinje().tidslinje.let {
-                when {
-                    it.isEmpty() -> KanStansesEllerGjenopptas.INGEN
-                    it.last() is UtbetalingslinjePåTidslinje.Stans -> KanStansesEllerGjenopptas.GJENOPPTA
-                    else -> KanStansesEllerGjenopptas.STANS
+                if (it.isNotEmpty() && ingenOpphørEllerHull(it)) {
+                    if (it.last() is UtbetalingslinjePåTidslinje.Stans) {
+                        KanStansesEllerGjenopptas.GJENOPPTA
+                    } else {
+                        KanStansesEllerGjenopptas.STANS
+                    }
+                } else {
+                    KanStansesEllerGjenopptas.INGEN
                 }
             },
             revurderinger = revurderinger.map { it.toJson() },
             vedtak = vedtakListe.map { it.toJson() },
-            klager = klager.map { it.toJson() }
+            klager = klager.map { it.toJson() },
         )
+
+        /**
+         * Tillater ikke stans dersom stønadsperiodene inneholder opphør eller hull, siden det kan bli problematisk
+         * hvis man stanser og gjenopptar på tvers av nevnte opphør og hull. Ta opp igjen problemstillingen dersom
+         * fag trenger å stanse på et slikt tilfelle.
+         */
+        private fun ingenOpphørEllerHull(utbetalingslinjer: List<UtbetalingslinjePåTidslinje>): Boolean {
+            if (utbetalingslinjer.any { it is UtbetalingslinjePåTidslinje.Opphør }) {
```

Comment:
Tilsvarende som under, dette vl vel eliminere muligheten for å stanse ting i fremtiden dersom det på noe som helst tidspunkt i fortiden har eksistert et opphør?

Reply:
Helt generelt er vel stans/gjenoppta ferskvare, og bør være utledet av ytelsen som eventuelt løper akkurat nå. Historiske tilstander på saken er vel i utgangspunktet ikke relevant.

Reply:
@jacob-meidell Stemmer, dette er en løsning som bare forhindrer nåværende potensielle feilsituasjoner, men som da også sperrer for muligheten til å stanse ved noen mulige fremtidige caser. Ønsker i utgangspunktet å bruke minst mulig tid på å undersøke corner cases her, så fag er happy med begrensningen, og at vi heller plukker den opp igjen dersom behovet for å stanse på en sak hvor muligheten er fjernet dukker opp.